cmd/asset-syncer/utils.go

```diff
@@ -145,6 +148,68 @@ func (r *HelmRepo) Repo() *models.RepoInternal {
 	return r.RepoInternal
 }
 
+func satisfy(chartInput map[string]interface{}, rule *apprepov1alpha1.FilterRuleSpec) (bool, error) {
+	query, err := gojq.Parse(rule.JQ)
+	if err != nil {
+		return false, fmt.Errorf("Unable to parse JQuery: %v", err)
+
+	}
+	varNames := []string{}
+	varValues := []interface{}{}
+	for name, val := range rule.Variables {
+		varNames = append(varNames, name)
+		varValues = append(varValues, val)
+	}
+	code, err := gojq.Compile(
+		query,
+		gojq.WithVariables(varNames),
+	)
+	if err != nil {
+		return false, fmt.Errorf("Unable to compile JQuery: %v", err)
```

Comment:
jq != JQuery

Reply:
ookay, thought that `jq query` was redundant but it's true that jquery is something else